plugins/modules/redfish_command.py

```diff
@@ -216,6 +216,35 @@
       - Handle to check the status of an update in progress.
     type: str
     version_added: '6.1.0'
+  update_custom_oem_header:
+    required: false
+    description:
+      - Optional OEM header, sent as separate form-data for
+        the Multipart HTTP push update.
+      - The header shall start with "Oem" according to DMTF
+        Redfish spec 12.6.2.2.
```

Comment:
I would suggest adding a link to that spec, like you did in the PR description.

Reply:
Done. Thanks for the review!